packages/store-ui/src/atoms/List/List.tsx

```diff
@@ -0,0 +1,56 @@
+import type { HTMLAttributes, ElementType } from 'react'
+import React, { forwardRef } from 'react'
+
+export type ListVariant = 'default' | 'description' | 'ordered' | 'unordered'
+
+const variantToType = (variant: ListVariant) => {
+  switch (variant) {
+    case 'description':
+      return 'dl'
+
+    case 'unordered':
+      return 'ul'
+
+    case 'ordered':
+      return 'ol'
+
+    default:
```

Comment:
Uhm, I'm a bit skeptical about the default rendered HTML element being a `div`. It's kind of an anti-pattern. 

How about making the default a `ul` tag? This feels more natural to me, and would probably be a more desired (and common) behavior. Since we're providing the `as` prop, if the user really wants to use a `<div>` here, he may do it.

Reply:
Yeah, it makes sense, the idea for a `div` was that it would be a "naked" list